src/tasks/daily/daily_battle_mixin.py

```
@@ -399,41 +432,84 @@ def _gather_retry_navigate(self, stage_name, category_name):
+            if is_extra_mode:
+                # 额外刷取则不领取奖励
+                cnt_extra_run += 1
+            else:
+                # 在『有可领取的奖励』页面上领取奖励
+                left_ticket = self.get_claim(stages_cost[category_name], left_ticket)
+                #
+                self.sleep(2)
+                if left_ticket <= 0:
+                    self.wait_click_ocr(match=re.compile("离开"), box=self.box.bottom_right, log=True, recheck_time=1)
+
+                    # 如果体力耗尽但设置了额外刷取次数，则开始额外刷取
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)